cmd/controller/controller.go

```diff
@@ -99,11 +106,212 @@ func NewController(store *store.AuthStore) *Controller {
 
 func (c *Controller) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	c.log.Info("Reconciling for", "name", req.NamespacedName.Name)
+
+	fromToForKey := req.NamespacedName.Namespace
+
+	crcSubjects := []v1a1.Subject{}
+	crcList := &v1a1.ClusterReferenceConsumerList{}
+	err := c.crClient.List(ctx, crcList)
+	if err != nil {
+		c.log.Error(err, "could not list ClusterReferenceConsumers")
+		return ctrl.Result{}, err
+	}
+
+	for _, crc := range crcList.Items {
+		for _, ref := range crc.References {
+			origin := fmt.Sprintf("%s/%s", ref.From.Group, ref.From.Resource)
+			target := fmt.Sprintf("%s/%s", ref.To.Group, ref.To.Resource)
+			key := fmt.Sprintf("%s;%s;%s", origin, target, ref.For)
```

Comment:
Nit: This seems like it could benefit from some kind of helper function(s) + type so we don't end up with some kind of accidental inconsistencies here.